improver/lapse_rate.py

```diff
@@ -290,80 +237,50 @@ def __repr__(self):
         )
         return desc
 
-    def _calc_lapse_rate(self, temperature, orography):
-        """Function to calculate the lapse rate.
-
-        This holds the function to determine the local lapse rate at a point by
-        calculating a least-squares fit to local temperature and altitude data
-        to find the local lapse rate.
+    def _create_windows(self, temp, orog):
+        """Uses neighbourhood tools to pad and generate rolling windows
+        of the temp and orog datasets.
 
         Args:
-            temperature (1D numpy.ndarray):
-                Contains the temperature values for the central point and its
-                neighbours.
-
-            orography (1D numpy.ndarray):
-                Contains the height values for the central point and its
-                neighbours.
+            temp:
+                The temperature dataset.
+            orog:
+                The orography dataset.
```

Comment:
The doc-strings are not currently formatted in the IMPROVER style. Please take a look at a few other plugins and make sure they conform.

Reply:
Done